internal/controller/ocivalidator_controller.go

```diff
@@ -55,9 +54,6 @@ func (r *OciValidatorReconciler) Reconcile(ctx context.Context, req ctrl.Request
 
 	validator := &v1alpha1.OciValidator{}
 	if err := r.Get(ctx, req.NamespacedName, validator); err != nil {
-		if !apierrs.IsNotFound(err) {
-			r.Log.Error(err, "failed to fetch OciValidator", "key", req)
-		}
```

Comment:
removed this to reduce some logs and to align more closely with patterns ive seen in the fluxcd/source-controller.

Reply:
lol, we've come full circle on this now

Reply:
lol i think in the initial PR we were always logging where fluxcd wasn't logging at all. the if statement got rid of some of the noise but not enough of it 🙃